doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -38,7 +38,7 @@ IDE, OS, and tool support
 Board support
 =============
 
-|no_changes_yet_note|
+* Added support for nRF7002-EB II Wi-Fi shield for use with nRF54L15 and nRF54LM20 DK boards.
```

Comment:
It is mentioned as nRF7002-EB II in https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/device_guides/nrf70/nrf7002eb2_dev_guide.html, so I suggest to write the same way.

@bama-nordic , Isn't is already mentioned for L15 in 3.0.0. https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/releases/release-notes-3.0.0.html#networking_samples?

Reply:
Quite right, sorry. But then again, https://docs.zephyrproject.org/latest/boards/shields/nrf7002eb/doc/index.html